doc/RELEASENOTES/CHANGELOG-1.0.4.md

```diff
@@ -0,0 +1,156 @@
+# CORTX OVA 1.0.4
+
+This release notes includes new features and bug fixes added to the CORTX OVA 1.0.4.
+
+# Features
+
+ 1. On the About page, the serial number is displaying.
```

Comment:
Can we change the sentence to below? As it is a first sentence in Release notes, it should be properly framed.
`Display serial number in About Page`.

Reply:
good catch, its corrected now

Reply:
looks like it is not refelcted yet.

Reply:
Please update the sentence to: 

`The serial number displays on the about page.`